CMakeLists.txt

```diff
@@ -114,6 +117,7 @@ if(ODBC)
   endif()
 endif()
 
+include(CMakeDependentOption)
 cmake_dependent_option(MYSQL "enable MathProg MySQL support" OFF "BUILD_SHARED_LIBS" OFF)
```

Comment:
should call it `WITH_MYSQL` for consistency ?

Reply:
GMP option removed, no longer inconsistent